config/rbac/role.yaml

```diff
@@ -297,88 +372,6 @@ rules:
   - patch
   - update
   - watch
-- apiGroups:
-  - ""
-  resources:
-  - clusterversions
-  - rhmis
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - ""
-  resources:
-  - configmaps
-  - events
-  - namespaces
-  - secrets
-  - secrets/finalizers
-  - serviceaccounts
-  - services/finalizers
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - ""
-  resources:
-  - configmaps/status
-  verbs:
-  - delete
-  - get
-  - patch
-  - update
-- apiGroups:
-  - ""
-  resources:
-  - deployments
-  - persistentvolumeclaims
-  - persistentvolumes
-  - pods
-  - pods/exec
-  - pods/log
-  verbs:
-  - '*'
-- apiGroups:
-  - ""
-  resources:
-  - endpoints
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - update
-  - watch
-- apiGroups:
-  - ""
-  resources:
-  - namespaces/finalizers
-  verbs:
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - ""
-  resources:
-  - services
-  verbs:
-  - '*'
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
```

Comment:
Only difference in the moving of this section of the file up to an earlier part of the file, is the removal of these verb entries. This should be fine given that they're here superfluously, given the first entry is `'*'`